accounts/abi/bind/bind_test.go

```diff
@@ -379,26 +394,26 @@ var bindTests = []struct {
 		`
 			"math/big"
 			"reflect"
-
-			"github.com/ethereum/go-ethereum/accounts/abi/bind"
-			"github.com/ethereum/go-ethereum/accounts/abi/bind/backends"
-			"github.com/ethereum/go-ethereum/common"
-			"github.com/ethereum/go-ethereum/core"
-			"github.com/ethereum/go-ethereum/crypto"
+	
+			"github.com/klaytn/klaytn/accounts/abi/bind"
+			"github.com/klaytn/klaytn/accounts/abi/bind/backends"
+			"github.com/klaytn/klaytn/common"
+			"github.com/klaytn/klaytn/blockchain"
+			"github.com/klaytn/klaytn/crypto"
 		`,
 		`
 			// Generate a new random account and a funded simulator
 			key, _ := crypto.GenerateKey()
 			auth := bind.NewKeyedTransactor(key)
-			sim := backends.NewSimulatedBackend(core.GenesisAlloc{auth.From: {Balance: big.NewInt(10000000000)}}, 10000000)
-
+			sim := backends.NewSimulatedBackend(blockchain.GenesisAlloc{auth.From: {Balance: big.NewInt(10000000000)}})
+	
 			// Deploy a slice tester contract and execute a n array call on it
 			_, _, slicer, err := DeploySlicer(auth, sim)
 			if err != nil {
 					t.Fatalf("Failed to deploy slicer contract: %v", err)
 			}
 			sim.Commit()
-
+	
```

Comment:
What are these changes? Tabs or whitespaces?

Reply:
Those are tabs, and they are useless.. I removed it https://github.com/klaytn/klaytn/pull/744/commits/e86bfb2469fcaf1d710ada5c05f854402bf12d49